packages/flutter/test/material/refresh_indicator_test.dart

```diff
@@ -248,7 +248,7 @@ void main() {
     bool completed = false;
     tester.state<RefreshIndicatorState>(find.byType(RefreshIndicator))
       .show()
-      .then<Null>((Null value) { completed = true; });
+      .then<void>((_) { completed = true; });
```

Comment:
Rather than `_`, please use `void value` so that it's clear what the argument is.